api/config.go

```diff
@@ -31,7 +31,7 @@ type Config struct {
 	// (ex 1) or in milli units to reflect the number of CPU shares (ex 500m).
 	CPU map[string]interface{} `json:"cpu,omitempty"`
 	// Healthchecks are the healthchecks that the application uses.
-	Healthcheck map[string]*Healthcheck `json:"healthcheck,omitempty"`
+	Healthcheck map[string]*Healthchecks `json:"healthcheck,omitempty"`
```

Comment:
Is the map formatted like `map[<process>][<healthcheck type>]`?

Is so, could it possibly be documented in the comment above for people using the API?